src/site/content/en/blog/next-gen-css-2019/index.md

```diff
@@ -0,0 +1,349 @@
+---
+title: Next-generation web styling
+subhead: Get up to date on some of the exciting features in modern CSS.
+description: |
+  Get up to date on some of the exciting new CSS features that are already
+  supported in modern browsers—or soon will be!
+
+# A list of authors. Supports more than one.
+authors:
+  - adamargyle
+  - una
+
+date: 2019-12-05
+# Add an updated date to your post if you edit in the future.
+# updated: 2019-11-01
+
+hero: hero.png
+# You can adjust the fit of your hero image with this property.
+# Values: contain | cover (default)
+hero_fit: contain
+
+# You can adjust the position of your hero image with this property.
+# Values: top | bottom | center (default)
+# hero_position: bottom
+
+# You can provide an optional cropping of your hero image to be used as a
+# thumbnail. Note the alt text will be the same for both the thumbnail and
+# the hero.
+# thumbnail: thumbnail.jpg
+
+alt: Adam Argyle and Una Kravetz showing that they'll talk about 12 CSS features.
+
+# You can provide a custom thumbnail and description for social media cards.
+# Images should be 1200 x 630.
+# If no social thumbnail is provided then the post will attempt to fallback to
+# the post's thumbnail or hero from above. It will also reuse the alt.
+# social:
+#   google:
+#     title: A title for Google search card.
+#     description: A description for Google search card.
+#     thumbnail: google_thumbnail.jpg
+#     alt: Provide an alt for your thumbnail.
+#   facebook:
+#     title: A title for Facebook card.
+#     description: A description for Facebook card.
+#     thumbnail: facebook_thumbnail.jpg
+#     alt: Provide an alt for your thumbnail.
+#   twitter:
+#     title: A title for Twitter card.
+#     description: A description for Twitter card.
+#     thumbnail: twitter_thumbnail.jpg
+#     alt: Provide an alt for your thumbnail.
+
+tags:
+  - post # post is a required tag for the article to show up in the blog.
+  - css
+  - accessibility
+  - houdini
+  - layout
+  - ux
+---
+
+There are a _ton_ of exciting things happening in CSS right now—and
+many of them are already supported in today's browsers!
+Our talk at CDS 2019, which you can watch below,
+covers several new and upcoming features we thought should get some attention.
+
+This post focuses on the features you can use today,
+so be sure to watch the talk
+for a deeper discussion of upcoming features like Houdini.
+You can also find demos for all the features we discuss on our
+[CSS@CDS page](a.nerdy.dev/css-at-cds).
+
+{% YouTube '-oyeaIirVC0' %}
+
+## Contents
```

Comment:
I like that you all put in this ToC because since the sections are unrelated to each other it makes a lot of sense that readers might want to skip around